src/App.tsx

```diff
@@ -24,6 +24,24 @@ const App = (): JSX.Element => {
   const history = useHistory();
   const apolloExplorer = hooks.useObservableState(graphql.apolloExplorerClientInstance$);
 
+  // save to magicstore - remove after campaign @anukulpandey
```

Comment:
Needs to go to magicsquare service file as a method and called from here.
Also add function there that calls the magicstore api which will be used from around the app.